### PR TITLE
Initial commits of Lucy LDD

### PR DESCRIPTION
**Pull request by jordanpadams**
Follow-on from https://github.com/pds-data-dictionaries/ldd-lucy/pull/1, but now with LDDs auto-generated by GitHub Actions.

NOTE: First GitHub Action execution failed: https://github.com/pds-data-dictionaries/ldd-lucy/actions/runs/4547119575

### CODE REVIEW

**Comment by jordanpadams** [comment]
@cgobat @katecrombie your changes have been merged into this new branch, and the [LDD execution failed](https://github.com/pds-data-dictionaries/ldd-lucy/actions/runs/4547119575) due to the test files still being the same as the template repo. I renamed those so they are no longer picked up by the GitHub action. and [manually re-launched it successfully](https://github.com/pds-data-dictionaries/ldd-lucy/actions/runs/4547198584). you should see the LDDs now generated on that branch. once you accept the invite to join the team, you will be able to merge as you see fit.

**Comment by cgobat** [comment]
Roger that. Thanks @jordanpadams!

File: README.md [hunk 1]
```diff
@@ -55,29 +54,29 @@ are located at https://pds-data-dictionaries.github.io/{repo id}.
        * Review the [Change Log] (ChangeLog.md)
        * Review the [Change Log] (src/CHANGELOG.md)
 -->     
-* Download dictionary files from the [PDS Data Dictionaries Page](https://pds.nasa.gov/datastandards/dictionaries/#{dictionary prefix})
+* Download dictionary files from the [PDS Data Dictionaries Page](https://pds.nasa.gov/datastandards/dictionaries/#lucy)
 * On [GitHub](../../releases/latest)
-* Review the [Change log]({change log file name})
+* Review the [Change log](CHANGELOG.md)
```
Comment (cgobat) [inline README.md:59]: Doesn't exist but I think that's okay for now

File: logs/v1.19.0.0_execution_logs/validate_report_20230328200935.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  PASS: file:/home/runner/work/ldd-lucy/ldd-lucy/build/development/0519f7b224dc98caea093a9452453b511f3e0516/1.19.0.0/PDS4_LUCY_1J00_1000.xml
+        1 product validation(s) completed
+
+Summary:
+
+  0 error(s)
+  0 warning(s)
+
+  Product Validation Summary:
+    1          product(s) passed
```
Comment (cgobat) [inline logs/v1.19.0.0_execution_logs/validate_report_20230328200935.txt]: Passed without even a warning. Nice :sunglasses: :+1:

File: src/PDS4_LUCY_IngestLDD.xml [hunk 3]
```diff
@@ -0,0 +1,1711 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1I00.sch"
```
Comment (cgobat) [inline src/PDS4_LUCY_IngestLDD.xml:2]: Note the use of the IM `1I00` schema here

File: logs/v1.19.0.0_execution_logs/lddtool_run_log_20230328200928.txt [hunk 4]
```diff
@@ -0,0 +1,36 @@
+
+>>  INFO LDDTOOL Version: 14.1.2
+>>  INFO IM Version Id: 1.19.0.0
```
Comment (cgobat) [inline logs/v1.19.0.0_execution_logs/lddtool_run_log_20230328200928.txt]: We have been developing to IM version 1.18.0.0 (`1I00`; see comment in `PDS4_LUCY_IngestLDD.xml`) so far. I see that the GitHub build action is set up to use 1.19.0.0 (`1J00`). Is this configurable? @katecrombie, do we care?

**Review by cgobat — COMMENTED** [review]
Looks good to me for the most part. Just want to make sure we're okay on the information model version discrepancy.

File: logs/v1.19.0.0_execution_logs/lddtool_run_log_20230328200928.txt (hunk 4 above)
Comment (jordanpadams) [reply logs/v1.19.0.0_execution_logs/lddtool_run_log_20230328200928.txt]: @cgobat the version of the PDS4 IM used in your IngestLDD vs. the version output from LDDTool are 2 different things, and do not need to be the same. to update the PDS4 version for the output schema/schematron. Update the `pds4_versions.txt` file at the top level of your repo. This versions should match the PDS4 version you plan to use for your project.

Comment (cgobat) [reply logs/v1.19.0.0_execution_logs/lddtool_run_log_20230328200928.txt]: Ah, I had missed that. Thanks.

**Review by katecrombie — APPROVED** [review]

**Comment by katecrombie** [comment]
Merged the initial lucy ldd to main.  Go lucy!

**Pull request merged**